test-integration/operator-tests/src/test/java/org/infinispan/operator/installation/MonitoringStackIT.java

```diff
@@ -69,15 +71,15 @@ static void prepare() throws IOException {
         grafanaShift.recreateProject();
         grafanaShift.operatorHub().operatorGroups().create(operatorGroup);
         grafanaShift.operatorHub().subscriptions().create(grafanaSubscription);
+        grafanaWaiter.areExactlyNPodsRunning(1, "name", "grafana-operator").waitFor();
```

Comment:
@Crumby have you validated with latest (3.10.1) Grafana operator version?

Reply:
Yes. Works as expected. But the tests themself verify only that Dashboard CR gets created. If you'd like to verify that dashboard works and looks fine you need to go and look